### DISCUSSION TITLE
Picture is not displayed after uploading to S3 (minio) storage

### DISCUSSION THREAD

**Discussion by quickstar** [outline/outline#6511] [Self Hosting] [answered]
**To Reproduce**
1. Open a page
2. Paste an image (e.g. screen shot) from clipboard to a page
3. The upload to the S3 storage is successfull (the request succeeds and the file is in the bucket)
4. A javascript error occurs preventing the image to be diplayed

**Expected behavior**
The image is shown in the document

**Screenshots**
![image](https://github.com/outline/outline/assets/1195919/adcd40cc-ad0a-4b1c-af6b-149a9f220c10)
It seems that the URL is not concated correctly. Our minio is at s3.domain.com and not outline.s3.domain.com

**Outline (please complete the following information):**
- Install: self hosted
- Version: v0.74.0

**Desktop (please complete the following information):**
 - Browser: Tested with latest versions of firefox and chrome

**Comment by linear** [comment]
<p><a href="https://linear.app/outline/issue/OLN-247/picture-is-not-displayed-after-uploading-to-s3-storage">OLN-247 Picture is not displayed after uploading to S3 storage</a></p>

**Comment by quickstar** [comment, marked as answer]
Setting the environment variable `AWS_S3_FORCE_PATH_STYLE=true` did the trick
